src/elements/content-sidebar/SidebarContent.scss

```diff
@@ -11,8 +11,8 @@
         align-items: center;
         justify-content: space-between;
         height: 60px;
-        margin: 0 25px;
```

Comment:
This change seems quite risky, as it will affect every child sidebar (Activity, Versions, Skills, Metadata, etc.) and is not behind a feature flip. Let's find a safer way to achieve the experience we want.